ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/checkbox/v1/checkbox/checkbox.html

```diff
@@ -32,17 +32,21 @@
      data-cmp-data-layer="${checkbox.data.json}"
      data-sly-test.widgetId="${'{0}-{1}' @ format=[component.id, '_widget']}"
      data-cmp-adaptiveformcontainer-path="${formstructparser.formContainerPath}">
-    <div data-sly-call="${questionMark.questionMark @componentId=checkbox.id, longDescription=checkbox.description, bemBlock='cmp-adaptiveform-checkbox'}" data-sly-unwrap></div>
+     <div class="cmp-adaptiveform-checkbox__label-container">
```

Comment:
how is this a abel container which doesn't have label?

Reply:
changed